ccx_keys/tests/test_ccx_keys.py

```diff
@@ -309,6 +309,7 @@ def test_missing_block_id(self, url):
         ('org', 'course', 'run', '1', 'category', 'name:more_name', None),
     )
     @ddt.unpack
+    # pylint: disable=too-many-positional-arguments
     def test_valid_locations(self, org, course, run, ccx, category, name, revision):
```

Comment:
```suggestion
    def test_valid_locations(self, org, course, run, ccx, category, name, revision):  # pylint: disable=too-many-positional-arguments
```

Reply:
Placing the `pylint` disable above the function would be better here imo because the function has a long signature with seven arguments. Adding an inline comment would crowd the line, making it harder to read. An above-line comment maintains clarity and improves readability for reviewers and maintainers.

Reply:
Fixed it in the upgrade requirements PR. You can simply rebase the PR to get it resolved
https://github.com/openedx/ccx-keys/commit/5c7cef1732a2a40d01f4b14beb22fc7ea9ac93a4#diff-4a1e5a86c548b14ce47668a70a21cf73c0d23eb4fb780ad87e2bc3f1e596ed9fR308